apps/ui/src/composables/useAiSummary.ts

```diff
@@ -0,0 +1,34 @@
+const SIDEKICK_URL = 'https://sh5.co';
+
+export function useAiSummary(proposalId: string) {
```

Comment:
Is this file used?

Reply:
No, seems like a botched merge from master